src/FixerConfiguration/FixerConfigurationResolver.php

```diff
@@ -62,6 +62,10 @@ public function resolve(array $options)
                 $alias = $option->getAlias();
 
                 if (array_key_exists($alias, $options)) {
+                    if (array_key_exists($name, $options)) {
+                        throw new \LogicException(sprintf('Aliased option %s/%s is passed multiple times.', $name, $alias));
```

Comment:
too late for me to check it, but it shall be `InvalidFixerConfigurationException` or `InvalidOptionsException`

Reply:
I couldn't throw `InvalidFixerConfigurationException` from here because I need access to the fixer name which I don't believe is possible in here. Throwing symfony's `InvalidOptionsException` feels a little hacky, but it means it's upgraded to a `InvalidFixerConfigurationException` by `AbstractFixer::configure`